train.py

```diff
@@ -138,6 +145,7 @@ def main(args):
     parser.add_argument('--save_path', type=str, default = "./checkpoints/", help="Path to the dataset")
     parser.add_argument('--num_epochs', type=int, default=50, help="Number of epochs to train")
     parser.add_argument('--batch_size', type=int, default=16, help="Mini batch size")
+    parser.add_argument('--test_interval', type=int, default=10, help="interval of test session during training")
```

Comment:
Aight this is enough for now. Let me know if that's it so that I can immediately merge your work.

Reply:
Sure. Also to add, default = 10 means the testing session will run 10 times for each epochs. For you it might be 1 test session for each 109 batches

Reply:
> Sure. Also to add, default = 10 means the testing session will run 10 times for each epochs. For you it might be 1 test session for each 109 batches

Would you more elaborate on this? I thought it is literally **interval** - if I set it to N, log printing for test set is carried out every Nth iteration. But what you mean by `default = 10 means the testing session will run 10 times for each epochs.`? You mean N is equal to the total number of test session within one epoch?

Reply:
line:52  test_interval = int(len(train_dataloader)/args.test_interval)
here I set test_interval. So, if you have 1000 iteration for training and set args.test_interval to 10, testing session will appear for each 100 (=1000/10) iterations

Reply:
Gotcha. Beautiful!